src/main/java/com/authlete/jaxrs/AccessTokenValidator.java

```diff
@@ -159,6 +159,54 @@ public AccessTokenInfo validate(String accessToken, String[] requiredScopes) thr
      */
     public AccessTokenInfo validate(
             String accessToken, String[] requiredScopes, String requiredSubject, String clientCertificate) throws WebApplicationException
+    {
+        return validate(accessToken, requiredScopes, requiredSubject, clientCertificate, null, null, null);
+    }
+
+
+    /**
+     * Validate an access token.
+     *
+     * </p>
+     * When the given access token is not valid, this method throws a
+     * {@link WebApplicationException}. The response contained in the
+     * exception complies with the requirements described in <a href=
+     * "http://tools.ietf.org/html/rfc6750">RFC 6750</a> (The OAuth
+     * 2.0 Authorization Framework: Bearer Token Usage).
+     * </p>
+     *
+     * @param accessToken
+     *            An access token to validate.
+     *
+     * @param requiredScopes
+     *            Scopes that must be associated with the access token.
+     *            {@code null} is okay.
+     *
+     * @param requiredSubject
+     *            Subject (= user's unique identifier) that must be associated
+     *            with the access token. {@code null} is okay.
+     *
+     * @param clientCertificate
+     *            TLS Certificate of the client presented during a call to
+     *            the resource server, used with TLS-bound access tokens.
+     *            Can be {@code null} if no certificate is presented.
+     *
+     * @return
+     *         Information about the access token.
+     *
+     * @throws WebApplicationException
+     *             The access token is invalid. To be concrete, one or more of
+     *             the following conditions meet.
+     *             <ol>
+     *             <li>The access token does not exist.
+     *             <li>The access token has expired.
+     *             <li>The access token does not cover the required scopes.
+     *             <li>The access token is not associated with the required subject.
+     *             </ol>
+     */
+    public AccessTokenInfo validate(
+            String accessToken, String[] requiredScopes, String requiredSubject, String clientCertificate,
+            String dpopHeader, String htm, String htu) throws WebApplicationException
```

Comment:
In authlete-java-jaxrs 2.25, I added `AuthorizationDecisionHandler.Params` class to pack many parameters into one class in order to avoid increasing the number of `handle` method variants disorderly. Probably, it's high time to create `AccessTokenValidator.Params` class or something equivalent.